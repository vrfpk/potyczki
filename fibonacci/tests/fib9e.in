53016200190744168
