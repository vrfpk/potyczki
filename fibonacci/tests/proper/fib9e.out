224999999999989326
